src/ui/pages/slack_alerts_setup/form.jsx

```diff
@@ -12,73 +15,166 @@ import { ButtonGroup, FormWrapper } from './styles';
-                    />
-                </ControlGroup>
-            </FormWrapper>
-            <Heading level={3}>Message Appearance</Heading>
-            <Paragraph>
-                The following settings will influence how messages will show up in Slack.{' '}
-                <Link onClick={() => openSlackMessagePreview(data)} openInNewContext>
-                    Show Preview
-                </Link>
-                .
-            </Paragraph>
-            <FormWrapper>
-                <ControlGroup
-                    label="Sender Name"
-                    help="This name will appear in slack as the user sending the message."
-                >
-                    <Text value={fromUserName} onChange={updateUser} disabled={loading} />
-                </ControlGroup>
-                <ControlGroup
-                    label="Sender Icon"
-                    help="The avatar/icon shown by the sender of the slack message. This URL needs to be accessible from the internet."
-                >
-                    <Text value={fromUserIcon} onChange={updateUserIcon} disabled={loading} />
-                </ControlGroup>
-            </FormWrapper>
-            <ButtonGroup>
-                <div>
-                    <Button label="Cancel" appearance="secondary" onClick={redirectToAlertListingPage} />
-                </div>
-                <div>
-                    <Button label="Save" appearance="primary" disabled={loading || !isDirty} onClick={save} />
-                </div>
-            </ButtonGroup>
+                            Incoming Webhooks
+                        </Link>{' '}
+                        to post messages from Splunk into Slack channels. You can set a default webhook URL here,
+                        which will be used for all alerts by default. Each alert can override and use a different
+                        webhook URL that has different permissions or can send to a different Slack workspace.
+                    </Paragraph>
+                    <FormWrapper>
+                        <ControlGroup
+                            label="Webhook URL"
+                            help={
+                                <Link to="https://slack.com/apps/A0F7XDUAZ-incoming-webhooks" openInNewContext>
+                                    Configure Slack incoming webhook
+                                </Link>
+                            }
+                        >
+                            <Text
+                                value={webhookUrl}
+                                onChange={updateUrl}
+                                disabled={loading}
+                                placeholder="https://hooks.slack.com/services/XXXXX/YYYY/ZZZZZ"
+                            />
+                        </ControlGroup>
+                    </FormWrapper>
+                </TabLayout.Panel>
+            </TabLayout>
+
+            <div style={{ margin: 20 }}>
```

Comment:
@ziegfried - The sender name and icon show up regardless of the tab you choose. The `chat:write.customize` setting worked and I tested changing the name and icon. Thanks for mentioning that!